communication/slack-guidelines.md

```diff
@@ -1,90 +1,232 @@
+Join the `#slack-admins` channel or contact one of the admins in the
+[closest timezone][admins] via DM directly and describe the situation. If the
+issue can be documented, please take a screenshot to include in your message.
 
-* Spreading spam content in DMs and/or channels
-* Not adhering to the code of conduct set forth in DMs and/or channels
-* Overtly selling products, related or unrelated to Kubernetes
+**What if you have a problem with an admin?**
 
-## SPECIFIC CHANNEL RULES
+Send a DM to another [listed admin][admins] and describe the situation. If it’s
+a [code of conduct][coc] issue, please send an email to <conduct@kubernetes.io>
+and describe the situation.
 
-In the case that certain channels have rules or guidelines, they will be listed in the purpose or pinned docs of that channel.
+---
 
-#kubernetes-dev = questions and discourse around upstream contributions and development to kubernetes
-#kubernetes-careers = job openings for positions working with/on/around Kubernetes. Post the job once and pin it. Pins expire after 30 days. Postings must include:  
-- A link to the posting or job description
-- The business name that will employ the Kubernetes hire
-- The location of the role or if remote is OK
+## Moderation
 
-## DM (Direct Message) Conversations
+### Admin Expectations and Guidelines
 
-Please do not engage in proprietary company specific conversations in the Kubernetes Slack instance. This is meant for conversations around related Kubernetes open source topics and community. Proprietary conversations should occur in your company Slack and/or communication platforms.  As with all communication, please be mindful of appropriateness, professionalism, and applicability to the Kubernetes community.
+Admins should adhere to the general Kubernetes project [moderation guidelines].
 
+Admins should additionally make sure to mention they are a Slack admin and their
+timezone in their “What I do” section of their Slack profile.
 
-Note:
-We archive the entire workgroup's slack data in zip files when we have time. [The latest archive is from June 2016-November 2018.](https://drive.google.com/drive/folders/1idJkWcDuSfs8nFUm-1BgvzZxCqPMpDCb?usp=sharing)
+Be mindful of how you handle communication during stressful interactions.
+Administrators act as direct representatives of the community, and need to
+maintain a very high level of professionalism at all times. If you feel too
+involved in the situation to maintain impartiality or professionalism, that’s
+a great time to enlist the help of another admin.
+
+Try to take any situations that involve upset or angry members to DM or video
+chat. Please document these interactions for other Slack admins to review.
+
+Content will be automatically removed if it violates code of conduct or is a
+sales pitch. Admins will take a screenshot of such behavior in order to document
```

Comment:
'will' or 'may'?

Reply:
I would stick with "will", but remove the "automatically" part.  Admins will do their best to remove any content that violates the CoC.